### PR TITLE
feat: Pbi 29 user timezone configuration backend

### PR DESCRIPTION
**Pull request by ferdifdi**
### Describe your changes
implement utc (add utc field)

### Backlog name
PBI-29-User_Timezone_Configuration-backend

### Acceptance criteria
- User can set timezone on user profile
- Date time shown on website is based on user timezone
- Alerts date time shown based on user timezone

### Checklist
- [X] I have performed a self-review of my code
- [X] Code successfully run on local
- [X] Feature / changes tested on local
- [X] No failing unit test
- [ ] Passed UAT Test
- [ ] Sonarqube tested 
- [ ] Required any changes to environment variable

### CODE REVIEW

File: apimonitor/models.py [hunk 1]
```diff
@@ -94,6 +94,11 @@ class AssertionExcludeKey(models.Model):
 
 class AlertsConfiguration(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
+
+    utc = models.IntegerField(default=0, validators=[
+        MinValueValidator(-1200),
```
Comment (luthfifahlevi) [inline apimonitor/models.py:99]: Ini kenapa batas minimumnya -1.200 dan maxnya 1.400 ya?
Bukannya utc minimal -12 dan max +12? Wkwk

Comment (ferdifdi) [reply apimonitor/models.py:99]: https://id.wikipedia.org/wiki/Daftar_zona_waktu. Dari situ, utc nya sampai +14. Kenapa gue nambahin 2 digit lebih karena ada UTC-9:30, UTC+5:45, dsb. Menurut lu gmn, Upi?

Comment (luthfifahlevi) [reply apimonitor/models.py:99]: oh okee berarti maxnya +14 dan minnya -12 aja fer. Kesepakatan kmrin kita implement per jam aja
Btw implementasi lu ngubah ke utc 0 ke +7 (misal), dimana ya?

File: apimonitor/migrations/0014_alertsconfiguration_utc.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="alertsconfiguration",
+            name="utc",
+            field=models.IntegerField(
+                default=0,
+                validators=[
+                    django.core.validators.MinValueValidator(-1200),
```
Comment (luthfifahlevi) [inline apimonitor/migrations/0014_alertsconfiguration_utc.py:20]: sebelum di push, di migrate dlu fer wkw hasil migratenya masih yg dlu nih wkwk

File: apimonitor/models.py (hunk 1 above)
Comment (ferdifdi) [reply apimonitor/models.py]: Sudah dipush lagi. Done

**Review by andrew-susanto — APPROVED** [review]

**Review by luthfifahlevi — APPROVED** [review]

**Pull request merged**